src/main/webapp/ts/login.ts

```diff
@@ -0,0 +1,28 @@
+window.onload = () => {
+  loadHome();
+};
+
+async function loadHome() {
+  const logging = document.getElementById('logging');
+  const link = document.getElementById('login-link');
+
+  const logStatus = await getLogStatus();
+
+  if (link && logging) {
+    if (logStatus.Bool) {
+      link.setAttribute('href', logStatus.Url);
```

Comment:
link.setAttribute and logging.style.display seem to be identical in this if/else statement. It's best to try to minimize repeated code.

Consider moving these outside the if/else statement if they do not depend on the logStatus.Bool condition.